euphonic/validate.py

```diff
@@ -45,15 +45,20 @@ def _check_constructor_inputs(
         if not isinstance(typ, list):
             typ = [typ]  # noqa: PLW2901 redefined-loop-name
         if not any(isinstance(obj, t) for t in typ):
-            raise TypeError(f"The type of {name} {type(obj)} doesn't "
-                             f'match the expected type(s) {typ}')
+            msg = (
+                f"The type of {name} {type(obj)} doesn't "
```

Comment:
```suggestion
                f"The type of {name} {type(obj).__name__} doesn't "
```
In future?

Reply:
Well, we are making small formatting changes to a few error messages here so it shouldn't hurt.